modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/internal/util/ItemSelectorRepositoryEntryBrowserUtil.java

```diff
@@ -244,10 +237,11 @@ private static void _addGroupSelectorBreadcrumbEntry(
 	}
 
 	private static void _addPortletBreadcrumbEntry(
-		long folderId, HttpServletRequest httpServletRequest, String title,
-		PortletURL portletURL) {
+		long folderId, HttpServletRequest httpServletRequest, long repositoryId,
+		String title, PortletURL portletURL) {
```

Comment:
This parameters should be sorted alphabetically.

Reply:
Done!